biomage/account/account.py

```diff
@@ -117,6 +115,11 @@ def _change_password(email, password, userpool):
     required=False,
     help="Password for the new account.",
 )
+@click.option(
+    "--userpool",
+    required=True,
+    help="Userpool to add the new account to.",
+)
```

Comment:
This is needed so that we can use the `biomage account user-create` command. Previously, there was no way of specifying the `userpool` parameter for the command.